v2.0/create-index.md

```diff
@@ -29,13 +33,14 @@ table td:first-child {
 | Parameter | Description |
 |-----------|-------------|
 |`UNIQUE` | Apply the [Unique constraint](unique.html) to the indexed columns.<br><br>This causes the system to check for existing duplicate values on index creation. It also applies the Unique constraint at the table level, so the system checks for duplicate values when inserting or updating data.|
+| `INVERTED` | <span class="version-tag">New in v2.0:</span> Create an [inverted index](inverted-indexes.html) on the specified column, which contains schemaless data (i.e., [`JSONB`](jsonb.html)).<br><br> You can also use the PostgreSQL-compatible syntax `USING GIN`. For more details, see [Inverted Indexes](inverted-indexes.html#creation).|
```

Comment:
Same comment as above: Shouldn't we just focus on `JSONB`?

Reply:
done